ch4/reversingAnArray.js

```diff
@@ -0,0 +1,47 @@
+const reverseArray = (inputArr) => {
+  let newArr = [];
+  for (let el of inputArr) {
+    newArr.unshift(el);
```

Comment:
out of curiosity, why use `unshift` instead of just `push`? [here's an SO ](https://stackoverflow.com/a/44032020) post about the performance profiles of `unshift` vs `push`.